HabitatSampler_v2.0/HabitatSampler_Usage.txt

```diff
@@ -4,7 +4,7 @@
 0) Load all required packages and functions
     0.1 set working directories (use: '/')
     0.2 set data input/output paths (use: '/')
-    0.3 automatic load and/or install libraries
+    0.3 install dependencies, HaSa R package and automatic load and/or install libraries
```

Comment:
libraries are loaded via the package...we should add a line library(HaSa)

Reply:
Do you meant to the documentation or to the R script?

Reply:
Covered by 47c83cd.